specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/blob.json

```diff
@@ -1024,6 +1069,38 @@
       ],
       "description": "The ImmutabilityPolicy property of a blob container, including Id, resource name, resource type, Etag."
     },
+    "VersionLevelWorm": {
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "This is an immutable property, when set to true enables version level worm at the container level."
+        },
+        "timeStamp": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "Returns the date and time the version level worm was enabled."
+        },
+        "migrationState": {
+          "type": "string",
+          "readOnly": true,
+          "enum": [
+            "InProgress",
+            "Completed",
+            "Abort"
+          ],
+          "x-ms-enum": {
+            "name": "MigrationState",
+            "modelAsString": true
+          },
+          "description": "This property denotes the container level worm to version level worm migration state."
+        }
+      },
+      "required": [
```

Comment:
Could we remove  "enabled" from required?
Like delete retension policy also don't make enabled as required: https://github.com/Azure/azure-rest-api-specs/blob/d26d0aed4ce4fec151a1b09bb26edb1fd2ae462d/specification/storage/resource-manager/Microsoft.Storage/stable/2021-01-01/common.json#L76

The current swagger spec will make the VLW "enabled" as a required parameter when create container.